community/modules/scheduler/schedmd-slurm-gcp-v6-controller/modules/slurm_files/scripts/load_bq.py

```diff
@@ -321,8 +321,16 @@ def main():
     # on failure, an exception will cause the timestamp not to be rewritten. So
     # it will try again next time. If some writes succeed, we don't currently
     # have a way to not submit duplicates next time.
+    print(f"loading BigQuery data in batches of size : {BQ_MAX_ROW_LOAD_SIZE}")
+    num_batches = (len(jobs) // BQ_MAX_ROW_LOAD_SIZE) + 1
```

Comment:
Looks like this logic will not work if 
```py
len(jobs) == BQ_MAX_ROW_LOAD_SIZE  => 
num_batches == 2 => 
start_job_idx goes out of bound of jobs list
```

While not a big deal , i suppose `bq_client.insert_rows` can handle empty `table` arg (please confirm if you want to keep current logic), I would advice to either:

* Fix `num_batches` computation, (e.g. use `math.ceil` or add `1` to the length);
* Write it without `num_batches`, e.g.:

```py
...
  while start_job_idx < len(jobs):
  ...
```

Reply:
whoops, fixed to a for loop